### PR TITLE
[BACK-2625] Do not send scheduled reports if the patient hasn't uploaded data recently

### PR DESCRIPTION
**Pull request by toddkazakov**

### CODE REVIEW

File: report/report.go
Comment (ewollesen) [inline report/report.go:1]: This change seems a bit odd... But I feel like I can assume you know what's going on here. :D

**Review by ewollesen — APPROVED** [review]
LGTM

File: report/report.go
Comment (toddkazakov) [reply report/report.go:1]: We don't want to send the exact same reports every two weeks if the user hasn't uploaded any new data. We even had a clinic which got confused by this.

Comment (toddkazakov) [reply report/report.go:1]: I merged the original PR before it was approved by accident and I reverted it.

**Pull request merged**